test/e2e/storage/csi_mock_volume.go

```diff
@@ -944,8 +945,194 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 			})
 		}
 	})
+
+	// These tests *only* work on a cluster which has the CSIStorageCapacity feature enabled.
+	ginkgo.Context("CSIStorageCapacity [Feature: CSIStorageCapacity]", func() {
```

Comment:
There shouldn't be a whitespace after "Feature".

Temporarily adding a whitespace in the test job so that we can get a run in today. https://github.com/kubernetes/test-infra/pull/18201

Reply:
Fixed.